lib/autorequire/data_import.rb

```diff
@@ -88,16 +88,19 @@ def self.import_topics
       language = Language.where("name like ?", "#{row["Topic_Language"]}%").first
       puts "Language #{row["Topic_Language"]} not found" unless language
       provider = Provider.find_by(id: row["Provider_ID"])
+      created_year = max(row["Created_Year"].to_i, 2016)
```

Comment:
Is max defined somewhere or dependent on a library? I'm getting an undefined method error.

Reply:
This method is defined on enums, I've updated the syntax